.github/workflows/run_iree_dispatch_profiler.yml

```diff
@@ -20,6 +20,7 @@ on:
     # Run daily at 2:00 PM (14:00) every weekday (Monday to Friday)
     - cron: "0 14 * * 1-5"
   workflow_dispatch:
+  pull_request: # TODO: Remove before submit
```

Comment:
don't forget to remove

Reply:
Keep this unresolved until the final commit just before the merge.

Reply:
removed `pull_request:`